packages/matchbox-site/src/components/SpacingPreview/SpacingPreview.js

```diff
@@ -1,20 +1,40 @@
 import React from 'react';
 import { meta } from '@sparkpost/design-tokens';
 import _ from 'lodash';
-import styles from './SpacingPreview.module.scss';
+import styled from 'styled-components';
+import { tokens } from '@sparkpost/design-tokens';
 
-const spacingScale = _.filter(meta, ({ name, value, category }) => !name.includes('base') && category === 'spacing' && value !== '0rem').reverse();
+import { Box } from '@sparkpost/matchbox';
+
+const StyledColumn = styled(Box)`
+  background-color: ${tokens.color_blue_700};
+  border-radius: ${tokens.borderRadius_100};
+  height: ${tokens.spacing_800};
+`;
+
+const spacingScale = _.filter(
```

Comment:
We added new spacing tokens recently, would you be able to update the package in this pr and see if this still works? I feel like we might need to omit a few